dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1945,6 +1945,28 @@ def put_student_in_section(student, teacher, script)
     assert extras_data['bonusLevels'][0]['levels'][0]['perfect']
   end
 
+  test "stage extras shows no progress if no current user" do
+    script = create :script
+    lesson_group = create(:lesson_group, script: script)
+    lesson = create(:lesson, script: script, lesson_group: lesson_group)
+    script_level = create :script_level, lesson: lesson, script: script
+    script_level.bonus = true
```

Comment:
Is there a reason you set bonus to true after creating the script level instead of creating it with bonus true like, `create :script_level, lesson: lesson, script: script, bonus: true`?